content/chapters/data/lab/support/memory-security/buff_leak.c

```diff
@@ -0,0 +1,84 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int a = 9;
+int b;
+
+int main()
+{
+    int c = 15;
```

Comment:
Use tabs, not spaces.

Reply:
that's a tab.

Reply:
```suggestion
	int c = 15;
```
No, it's not. This ^ is a tab (`\t`).

Reply:
It's a tab. but it's length 4 tab.

Reply:
Still no. You can copy and `xxd` this line:

```
$ echo "    int c = 15;" | xxd
00000000: 2020 2020 696e 7420 6320 3d20 3135 3b0a      int c = 15;.
```

The `0x20` characters are spaces. Tabs would be `0x09`. Alternatively, you can `xxd` the whole file for the same conclusion:

```
$ xxd buff_leak.c 
00000040: 6169 6e28 290a 7b0a 2020 2020 696e 7420  ain().{.    int 
```

And you can't create tabs of a specific length. That is only set locally. GitHub renders them with length 8.